fobi_custom/override_theme/templates/override_theme/edit_form_element_entry_ajax.html

```diff
@@ -0,0 +1,15 @@
+{% extends fobi_theme.form_edit_ajax %}
+
+{% load i18n %}
+
+{% block form_page_title %}
+{% blocktrans with form_element_plugin.name as plugin_name %}Edit survey question{% endblocktrans %}
+{% endblock form_page_title %}
+
+{% block form_id %}form_element_entry_form{% endblock %}
+
+{% block form_primary_button_text %}{% trans "Submit" %}{% endblock %}
+
+{% block form_secondary_button %}
+<a class="btn btn-warning" href="{% url 'fobi.edit_form_entry' form_entry.id %}">Cancel</a>
```

Comment:
Similar comment as above about button styles here.

Reply:
Looks great! Done for both `Cancel` buttons